devtools/gammacli/commands/run.go

```diff
@@ -100,7 +104,7 @@ func (r *CommandRunner) HandleRunCommand(args []string) (string, error) {
 			return "", err
 		}
 
-		if err := json.Unmarshal(jsonBytes, tx); err != nil {
+		if err = json.Unmarshal(jsonBytes, tx); err != nil {
```

Comment:
I don't understand this change

Reply:
i think again it was about overshadow as there is multiple use of err and its very unclear. i'd fix up the whole func if the owner would do it.

Reply:
sit with @itamararjuan and fix it?